README.md

```diff
@@ -1,21 +1,107 @@
-# Website
+# Get Started
 
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
-## Installation
+Docusaurus is a **static-site generator**. It builds a **single-page application** with fast client-side navigation, leveraging the full power of **React** to make your site interactive. It provides out-of-the-box **documentation features** but can be used to create **any kind of site** (personal website, product, blog, marketing landing pages, etc).
```

Comment:
Bit curious that we're basically advertising Docusaurus. I feel like this paragraph is unnecessary.

Reply:
Removed the unnecessary paragraph